### ISSUE TITLE
Is the gauge_settings.html file missing ?

### ISSUE THREAD

**Issue by VeryBitter** [m-labs/ionpak#13]
Hi
I am thinking of giving it a go to this gauge controller! Looks very promising (and cheap) to me). I work with MBE in an academic research centre. Do you have some estimation for the time it takes to solder all the components?

Looking at the software folder, it seems to me that the "gauge_settings.html" file is missing. Isn't it?

Great job you have done!

**Comment by sbourdeauducq** [comment]
Yep, it is missing. Thanks for your interest, but before you get too excited, I have to point out that this project is still in at an early stage of development and that I generally lack time to work on it. So, there are many important things that are currently missing, such as that page. If you're willing to help out, it would be great :)

**Comment by VeryBitter** [comment]
Thanks for the swift answer! I could try to help. But first I will have to test my soldering skills!
